src/OrchardCore/OrchardCore/Environment/Shell/RunningShellTable.cs

```diff
@@ -100,7 +100,7 @@ public ShellSettings Match(string host, string appRelativePath, bool fallbackToD
         private bool TryMatchInternal(string host, string appRelativePath, bool fallbackToDefault, out ShellSettings result)
         {
             // We can skip some checks for port matching if we know the host doesn't contain one
-            var hasPort = host.IndexOf(':') != -1;
+            var hasPort = host.Contains(':');
```

Comment:
Like @jtkech correctly pointed out it's fine to use Contains with netcoreapp3.0/nestandard2.1, it doesn't use LINQ